docs/reference/plugins.md

```diff
@@ -0,0 +1,82 @@
+# Plugins
+
+By default, the following WordPress plugins are installed with the latest version during the OCI
+image build time. If the plugins are installed during runtime with
+`juju config wordpress-k8s plugins=...`, the plugin will also be installed to it's latest version
+by default and may cause version differences between pods.
+Due to the clustered nature of the WordPress charm, installing plugins through UI has been disabled
+and can only be installed through the plugins configuration. Please see
+[Configurations](configurations.md) section for more information.
```

Comment:
Similar to earlier note, I'm guessing this reference won't work on charmhub

Reply:
Updated to the configure tab link: https://charmhub.io/wordpress-k8s/configure?channel=edge
Thank you for the comment!